src/app/content/components/Toolbar/styled.tsx

```diff
@@ -131,6 +133,12 @@ export const SearchButton = styled(({ desktop, mobile, ariaLabelId, ...props })
     <SearchIcon/>
   </PlainButton>;
 })`
+  height: 3.2rem;
+  background:
```

Comment:
does this look horrible when the flag loads and it suddenly changes to orange? maybe we could add a transition on the background color?

Reply:
It loads the same as the BookBanner color so I think we should either add a transition to both or neither.

Reply:
but book banner will have its color before the feature flags are loaded

Reply:
ah that's right. so far it looks fine but I'll add a .2s background transition.